src/components/ControlBar/EditBar.js

```diff
@@ -151,7 +187,9 @@ const EditBar = props => {
                     ? i18n.t('Exit Print preview')
                     : i18n.t('Print preview')}
             </Button>
-
+            <Button onClick={toggleFilterSettingsDialog}>
+                {i18n.t('Filter Settings')}
```

Comment:
Please use sentence case for buttons: `Filter settings`

Reply:
fixed